Volha_Ivanova/Module_4_Framework/Layers_Runner_BusinessObjects/my-app/src/main/java/com/epam/cdp/volha_ivanova/task1/pages/AccountMenu.java

```diff
@@ -0,0 +1,63 @@
+package com.epam.cdp.volha_ivanova.task1.pages;
+
+import org.openqa.selenium.Alert;
+import org.openqa.selenium.By;
+import org.openqa.selenium.NoAlertPresentException;
+import org.openqa.selenium.UnhandledAlertException;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+
+public class AccountMenu {
```

Comment:
AccountMenu is not a page. It's part of page. Please inject field accountMenu in appropriate page and add getter. 
page.getAccountMenu().logOut() for example